### PR TITLE
add dmonteiro caption

### PR DESCRIPTION
**Pull request by dmonteiro-ecsd**
This pull request is to add caption for dmonteiro.

@stoe all ok?

### CODE REVIEW

File: _posts/0000-01-03-dmonteiro-ecsd.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![panda-birthday](https://cloud.githubusercontent.com/assets/16547949/25400994/60f64804-29c2-11e7-8a6d-4401d7d294b9.jpg)
 {: .center}
 
-Caption here
+"Omg it's PANDA BIRTHDAY!"
```
Comment (recet) [inline _posts/0000-01-03-dmonteiro-ecsd.md:9]: Good work

**Review by recet — COMMENTED** [review]
Looks good

**Review by m1crobrain — APPROVED** [review]
looks good

**Review by hbayado — APPROVED** [review]
Looks awesome

**Pull request merged**